Updated meta descriptions flagged as 'too short' (#724)
MicrosoftDocs/quantum-docs-pr: articles/concepts/multiple-qubits.md

---
title: Multiple qubits

> description: Learn how to build, perform operations and measure two-qubit states. 

Reviewer: ```suggestion
description: Learn how to perform operations on registers with two or more qubits.
```

> description: Learn how to perform operations on registers with two or more qubits.

Reviewer 2: ```suggestion
description: Learn how to perform operations on two or more qubits.
```
